lib/command.js

```diff
@@ -185,6 +186,11 @@ module.exports = function (yargs, usage, validation) {
       if (demand.variadic) argv[demand.cmd] = argv._.splice(0)
       else argv[demand.cmd] = argv._.shift()
       postProcessPositional(yargs, argv, demand.cmd)
+      if (/-/.test(demand.cmd)) {
+        var cc = camelCase(demand.cmd)
```

Comment:
I think it might be worthwhile pulling this into a helper function, e.g., `addCamelcaseExpansions ()`? since the logic is the same in optional and required arguments.